tools.sh

```diff
@@ -22,18 +22,18 @@ ARCH=$(uname -m)
 case "${ARCH}" in
     x86_64)
         yq_ARCH="amd64"
-        SNC_GENERATE_MACOS_BUNDLE=1
-        SNC_GENERATE_WINDOWS_BUNDLE=1
+        SNC_GENERATE_MACOS_BUNDLE=${SNC_GENERATE_MACOS_BUNDLE:-1}
```

Comment:
I'd take a slightly different approach which only needs changes in tools.sh:  `SNC_DISABLE_MACOS_BUNDLE=1 ./snc.sh` and then
```
if [ -z ${SNC_DISABLE_MACOS_BUNDLE-} ]; then
    SNC_GENERATE_MACOS_BUNDLE=1
fi
```

Reply:
this mean it would be looks like in tools.sh? Also how do you set the default value like for `arm64` where we don't want to generate the windows bundle by default?

```
case "${ARCH}" in
    x86_64)
            if [ -z ${SNC_DISABLE_MACOS_BUNDLE-} ]; then
                   SNC_GENERATE_MACOS_BUNDLE=1
            fi
            if [ -z ${SNC_DISABLE_WINDOWS_BUNDLE-} ]; then
                   SNC_GENERATE_WINDOWS_BUNDLE=1
            fi
    [...]
          same for all the arch            
```

Reply:
I'd use something like this:
```
diff --git a/tools.sh b/tools.sh
index 5893404..44183a9 100755
--- a/tools.sh
+++ b/tools.sh
@@ -20,20 +20,25 @@ PATCH=${PATCH:-patch}
 ARCH=$(uname -m)
 
 case "${ARCH}" in
+    SNC_GENERATE_MACOS_BUNDLE=
+    SNC_GENERATE_WINDOWS_BUNDLE=
     x86_64)
         yq_ARCH="amd64"
-        SNC_GENERATE_MACOS_BUNDLE=1
-        SNC_GENERATE_WINDOWS_BUNDLE=1
+	if [ -z ${SNC_DISABLE_MACOS_BUNDLE-} ]; then
+            SNC_GENERATE_MACOS_BUNDLE=1
+	fi
+	if [ -z ${SNC_DISABLE_WINDOWS_BUNDLE-} ]; then
+            SNC_GENERATE_WINDOWS_BUNDLE=1
+	fi
 	;;
     aarch64)
         yq_ARCH="arm64"
-        SNC_GENERATE_MACOS_BUNDLE=1
-        SNC_GENERATE_WINDOWS_BUNDLE=
+	if [ -z ${SNC_DISABLE_MACOS_BUNDLE-} ]; then
+            SNC_GENERATE_MACOS_BUNDLE=1
+	fi
 	;;
     *)
         yq_ARCH=${ARCH}
-        SNC_GENERATE_MACOS_BUNDLE=
-        SNC_GENERATE_WINDOWS_BUNDLE=
 	;;
 esac
 ```

Reply:
@cfergeau even with this if now I need to generate the windows bundle for `aarch64` I need to make change in the script because the by default only I can only enable or disable macos bundle not windows one. The current PR actually allow user to enable/disable win/mac bundles for any arch without losing the default values.

Reply:
Grmble, the PR description is:
> Right now if user don't want to generate mac/win bundles for x86_64 arch then changes in the script needed. With this PR user can use following way in case don't want to generate any of those bundles.

This says nothing about force-enabling generation of disabled bundles. My reason for suggesting alternatives is that one must remember that the magic values to use are `0` and `1`, not `yes` and `no`, or `enabled` and `disabled`.

Do you expect to often need to enable/disable windows bundle generation on the fly on aarch64 that changing the code is going to get in the way?

Reply:
> Grmble, the PR description is:

My bad, will update the description and not to use any specific arch. I used x86_64 as example but it created the confusion :(

> My reason for suggesting alternatives is that one must remember that the magic values to use are 0 and 1, not yes and no, or enabled and disabled.

Yes, I also thought that but again these exposed variable we should document so user can look beforehand https://github.com/crc-org/snc/issues/431 (old issue)

> Do you expect to often need to enable/disable windows bundle generation on the fly on aarch64 that changing the code is going to get in the way?

Think about if I want to enable the CI to get some experimental bundles of windows for aarch64, I can just set the variable accordingly and don't care about changing the code. If I want to generate something manually I can always change the code according.

Reply:
> Yes, I also thought that but again these exposed variable we should document so user can look beforehand

The latest version of the PR is missing the documentation.